src/api/v4/layer/aggregation.js

```diff
@@ -0,0 +1,150 @@
+var _ = require('underscore');
+var CartoValidationError = require('../error-handling/carto-validation-error');
+
+// Taken from https://carto.com/docs/carto-engine/maps-api/tile-aggregation/#columns
+var VALID_OPERATIONS = {
```

Comment:
We must provide an enum for this, just like we do with `carto.dataview.status`, `carto.dataview.status`, ...